apps/vr-tests-react-components/src/stories/ProgressBar.stories.tsx

```diff
@@ -1,20 +1,43 @@
 import * as React from 'react';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=27156&commitId=51f9a52a1b3402cf8a646623810730c2023e28e2&env=prod&clientType=FLUENTUI)


  <details><summary><b>ProgressBar converged</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ProgressBar converged.Warning.chromium.png|1339| Changed|
|ProgressBar converged.Success.chromium.png|1491| Changed|
|ProgressBar converged.Error.chromium.png|1343| Changed|
|ProgressBar converged.Determinate + thickness - Dark Mode.chromium.png|0| Added|
|ProgressBar converged.Determinate + thickness - High Contrast.chromium.png|0| Added|
|ProgressBar converged.Determinate + thickness - RTL.chromium.png|0| Added|
|ProgressBar converged.Determinate + thickness.chromium.png|0| Added|
|ProgressBar converged.Indeterminate + thickness - Dark Mode.chromium.png|0| Added|
|ProgressBar converged.Indeterminate + thickness - High Contrast.chromium.png|0| Added|
|ProgressBar converged.Indeterminate + thickness - RTL.chromium.png|0| Added|
|ProgressBar converged.Indeterminate + thickness.chromium.png|0| Added|
|ProgressBar converged.Determinate - Dark Mode.chromium.png|0| Removed|
|ProgressBar converged.Determinate - High Contrast.chromium.png|0| Removed|
|ProgressBar converged.Determinate - RTL.chromium.png|0| Removed|
|ProgressBar converged.Determinate with thickness large.chromium.png|0| Removed|
|ProgressBar converged.Determinate.chromium.png|0| Removed|
|ProgressBar converged.Indeterminate - Dark Mode.chromium.png|0| Removed|
|ProgressBar converged.Indeterminate - High Contrast.chromium.png|0| Removed|
|ProgressBar converged.Indeterminate - RTL.chromium.png|0| Removed|
|ProgressBar converged.Indeterminate with thickness large.chromium.png|0| Removed|
|ProgressBar converged.Indeterminate.chromium.png|0| Removed|

</details>


  <details><summary><b>Skeleton converged</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Skeleton converged.Opaque Skeleton with rectangle - RTL.chromium.png|133| Changed|
|Skeleton converged.Opaque Skeleton with circle - Dark Mode.chromium.png|24985| Changed|
|Skeleton converged.Opaque Skeleton with square - High Contrast.chromium.png|29771| Changed|
|Skeleton converged.Opaque Skeleton with rectangle - Dark Mode.chromium.png|23915| Changed|
|Skeleton converged.Opaque Skeleton with circle - RTL.chromium.png|16| Changed|
|Skeleton converged.Translucent Skeleton with  circle - RTL.chromium.png|25| Changed|
|Skeleton converged.Opaque Skeleton with square - RTL.chromium.png|23| Changed|
|Skeleton converged.Opaque Skeleton with rectangle - High Contrast.chromium.png|28496| Changed|
|Skeleton converged.Translucent Skeleton with  circle - Dark Mode.chromium.png|24985| Changed|
|Skeleton converged.Opaque Skeleton with square - Dark Mode.chromium.png|24985| Changed|
|Skeleton converged.Opaque Skeleton with circle - High Contrast.chromium.png|29771| Changed|
|Skeleton converged.Translucent Skeleton with rectangle - Dark Mode.chromium.png|23915| Changed|
|Skeleton converged.Translucent Skeleton with  circle - High Contrast.chromium.png|29771| Changed|
|Skeleton converged.Translucent Skeleton with rectangle - RTL.chromium.png|203| Changed|
|Skeleton converged.Translucent Skeleton with rectangle - High Contrast.chromium.png|28496| Changed|
|Skeleton converged.Translucent Skeleton with square - Dark Mode.chromium.png|24985| Changed|
|Skeleton converged.Translucent Skeleton with square - High Contrast.chromium.png|29771| Changed|
|Skeleton converged.Translucent Skeleton with square - RTL.chromium.png|35| Changed|

</details>


  <details><summary><b>Spinner converged</b> 87 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Spinner converged.Primary - High Contrast.chromium.png|8542| Changed|
|Spinner converged.Inverted.chromium.png|8338| Changed|
|Spinner converged.Inverted - High Contrast.chromium.png|12846| Changed|
|Spinner converged.Inverted - Dark Mode.chromium.png|8338| Changed|
|Spinner converged.Inverted with Label - Dark Mode.chromium.png|46837| Changed|
|Spinner converged.Primary with Label.chromium.png|2204| Changed|
|Spinner converged.Inverted with Label - High Contrast.chromium.png|72157| Changed|
|Spinner converged.Primary with Label - High Contrast.chromium.png|69067| Changed|
|Spinner converged.Primary - Dark Mode.chromium.png|7168| Changed|
|Spinner converged.Primary with Label - Dark Mode.chromium.png|58391| Changed|
|Spinner converged.Inverted with Label.chromium.png|46837| Changed|
|Spinner converged.Primary.chromium.png|363| Changed|
|Spinner converged.Inverted + size - Dark Mode.chromium.png|0| Added|
|Spinner converged.Inverted + size - High Contrast.chromium.png|0| Added|
|Spinner converged.Inverted + size.chromium.png|0| Added|
|Spinner converged.Inverted - RTL.chromium.png|0| Added|
|Spinner converged.Inverted with Label - RTL.chromium.png|0| Added|
|Spinner converged.Primary + size - Dark Mode.chromium.png|0| Added|
|Spinner converged.Primary + size - High Contrast.chromium.png|0| Added|
|Spinner converged.Primary + size - RTL.chromium.png|0| Added|
|Spinner converged.Primary + size.chromium.png|0| Added|
|Spinner converged.Primary with Label - RTL.chromium.png|0| Added|
|Spinner converged.Inverted with Huge - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Huge - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Huge.chromium.png|0| Removed|
|Spinner converged.Inverted with Label Above - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Label Above - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Label Above.chromium.png|0| Removed|
|Spinner converged.Inverted with Label After - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Label After - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Label After.chromium.png|0| Removed|
|Spinner converged.Inverted with Label Before - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Label Before - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Label Before.chromium.png|0| Removed|
|Spinner converged.Inverted with Label Below - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Label Below - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Label Below.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Extra Large - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Extra Large - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Extra Large.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Extra Small - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Extra Small - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Extra Small.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Large - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Large - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Large.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Medium - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Medium - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Medium.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Small - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Small - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Small.chromium.png|0| Removed|
|Spinner converged.Primary with Huge - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Huge - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Huge.chromium.png|0| Removed|
|Spinner converged.Primary with Label Above - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Label Above - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Label Above.chromium.png|0| Removed|
|Spinner converged.Primary with Label After - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Label After - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Label After - RTL.chromium.png|0| Removed|
|Spinner converged.Primary with Label After.chromium.png|0| Removed|
|Spinner converged.Primary with Label Before - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Label Before - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Label Before - RTL.chromium.png|0| Removed|
|Spinner converged.Primary with Label Before.chromium.png|0| Removed|
|Spinner converged.Primary with Label Below - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Label Below - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Label Below.chromium.png|0| Removed|
|Spinner converged.Primary with Size Extra Large - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Size Extra Large - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Size Extra Large.chromium.png|0| Removed|
|Spinner converged.Primary with Size Extra Small - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Size Extra Small - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Size Extra Small.chromium.png|0| Removed|
|Spinner converged.Primary with Size Large - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Size Large - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Size Large.chromium.png|0| Removed|
|Spinner converged.Primary with Size Medium - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Size Medium - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Size Medium.chromium.png|0| Removed|
|Spinner converged.Primary with Size Small - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Size Small - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Size Small.chromium.png|0| Removed|
|Spinner converged.Primary with Size Tiny - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Size Tiny - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Size Tiny.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>